app/views/data-requirements.html

```diff
@@ -146,7 +148,7 @@ <h2 class='govuk-heading-s'>Non-UK degrees</h2>
         captionClasses: "govuk-visually-hidden",
         head: [
           {
-            text: "Specific data for Assessment Only",
+            text: "Specific data for assessment-only trainees",
```

Comment:
```suggestion
            text: "Specific data for assessment-only candidates",
```

I wonder if in this case we should use the correct term

Reply:
yes!